tdi_python/tdicli.py

```diff
@@ -62,83 +62,83 @@ def tdi_table_entry_get(tbl_hdl, session, dev_tgt, key, data, flag):
         setattr(self, 'tdi_table_entry_get', tdi_table_entry_get)
 
         def tdi_table_entry_get_by_handle(tbl_hdl, session, dev_tgt, ent_hdl, key, data, flag):
-            flags = c_uint64(flag)
+            flags = self.get_flags()
```

Comment:
How are we converting the flag input by user to the flags object? Shouldn't we be converting that? This doesn't look correct

Reply:
get_flags() is function to access previous created self._driver.tdi_flags_create(flag,  byref(self._flags)) and saved on  self._flags